duckduckgo/duckduckgo-privacy-extension
# remove unused social widget content script

**Reviewer:** @jdorweiler 
cc @russellholt 

## Description:
Remove calls to unused social-widgets content script.


## Steps to test this PR:
1. after reloading extension in chrome, visit any website and open devtools. Navigate to the 'sources' tab, and on the left navigate to the 'content scripts' tab. There shouldn't be anything there.
2. To really verify, try testing the code snippet feature on stackoverflow which was breaking due to frame access errors stemming from this content script. Click the 'run code snippet' button in the second answer here: https://stackoverflow.com/questions/50119240/why-this-recursive-javascript-function-returns-undefined
